README.md

```diff
@@ -41,3 +51,49 @@ To install HTT as a opam package from your local repository, run the [following
 ```
 opam install .
 ```
+
+## Supplementary material
+
+* [Partiality, State and Dependent Types](http://software.imdea.org/~aleks/htt/tlca11.pdf)
```

Comment:
Maybe we should put the doi links. They are more stable than those on @aleksnanevski  website.

Reply:
I think it's kind of logical that we link to IMDEA's site from IMDEA's github :) Also, some of DOI links are kind of broken, like this one https://dl.acm.org/doi/10.5555/2021953.2021972 points to Springer's site and lands on 404 page. If you look for the article yourself, it's there but behind a paywall. We could deal with this later, if Aleks' site goes down for some reason.

Reply:
the link works from my computer.
also, one problem with the current zip file on my page, is that it contains changes that aren't in the repo. the two should somehow be merged.

Reply:
The ACM link itself works, but the blue "Publisher Site" button doesn't. 

Regarding the zip, that's why I proposed adding some sort of a deprecation notice, or just link to this repo instead, because otherwise we'd have to manually merge it every time this repo is updated.

Reply:
right. the link to the repo is probably the right choice. but only *after* the extra code that the zip file contains is moved into the repo.